ac1/_sources/regression_and_line_of_best_fit/making_predictions_with_the_regression_line.rst

```diff
@@ -8,14 +8,20 @@
 Making Predictions with the Regression Line
 ===========================================
 
+Recall that in
+:ref:`(the section on regression lines)<regression_and_line_of_best_fit>`, you
+learned about the equation for the regression line.
+
 .. image:: figures/average_sat_score_completion_rate.png
 
 Returning to the scatter plot showing average SAT score and completion rate, the
 equation of the regression line is y = 0.0011*x - 0.7116. The slope is 0.0011
 and the y-intercept is -0.7116. Another way to write this is to use the
 descriptions of the x and y variables.
 
-(Predicted Completion Rate) = 0.0011 \* (Average SAT Score) - 0.7116
+
+:math:`(\text{Predicted Completion Rate}) = 0.0011 \cdot (\text{Average SAT
```

Comment:
This is likely personal preference, but I don't love \cdot as a multiplication notation. I prefer either nothing, or `*`. If y'all feel differently, I can be persuaded, I am not too fussed.

Reply:
I think an explicit multiplication symbol might be better because it's doing 0.0011 (Average SAT..) and students might mistake the parenthesized variable to be a comment instead of a variable? I mean obviously you can tell from context that 0.0011 is not a real SAT score but just to be safe. Did you mean the literal asterisk or something like \times?
![Screen Shot 2020-02-20 at 11 47 00 AM](https://user-images.githubusercontent.com/29069193/74972530-be5b9080-53d6-11ea-8f06-a02abe6fac0d.png)
![Screen Shot 2020-02-20 at 11 47 40 AM](https://user-images.githubusercontent.com/29069193/74972570-d4695100-53d6-11ea-9bc2-5c611a38b2be.png)
![Screen Shot 2020-02-20 at 11 48 05 AM](https://user-images.githubusercontent.com/29069193/74972594-e21ed680-53d6-11ea-9a55-ee7172141d44.png)

Reply:
![Screen Shot 2020-02-20 at 11 49 06 AM](https://user-images.githubusercontent.com/29069193/74972684-07abe000-53d7-11ea-8214-56fd8f8402e3.png)

Reply:
I like \times! Let's use that. It is very explicit and obvious.

@celinepark

Reply:
@raskutti Just changed the \cdot to \times and removed pavement.py. Not sure what's happening with the merge conflict

Reply:
@celinepark @lindseyc 

I don't think I can approve this PR until the pavement.py file is remvoed from this cl, not deleted. Atm, the file is deleted in this PR, so if I approve it, it will delete the master copy of this file.

Reply:
@raskutti I think I fixed the problem with pavement